docs/src/api/class-locatorassertions.md

```diff
@@ -2396,6 +2396,13 @@ await expect(page.locator('body')).toMatchAriaSnapshot({ name: 'body.aria.yml' }
 Name of the snapshot to store in the snapshot folder corresponding to this test.
 Generates sequential names if not specified.
 
+### option: LocatorAssertions.toMatchAriaSnapshot#2.exact
```

Comment:
When supporting the other overload, you'll have to handle it in the docs.

Reply:
Handled overload in docs